packages/react-vapor/src/components/table-hoc/TableWithBlankSlate.tsx

```diff
@@ -36,13 +36,13 @@ export const tableWithBlankSlate = (supplier: ConfigSupplier<IBlankSlateProps> =
         render() {
             const newProps = {
                 ..._.omit(this.props, [...TableWithBlankSlatePropsToOmit]),
-                renderBody: this.props.isEmpty ? (): any => null : this.props.renderBody,
             };
-            return (
-                <Component {...newProps}>
-                    {this.props.isEmpty ? <BlankSlate {...HocUtils.supplyConfig(supplier)} /> : this.props.children}
-                </Component>
-            );
+
+            if (_.isNull(this.props.data)) {

```

Comment:
You considered that this will return `false` if `this.props.data` is `undefined` ?